src/sentry/api/serializers/models/group.py

```diff
@@ -467,6 +467,15 @@ def serialize(self, obj, attrs, user):
             "hasSeen": attrs["has_seen"],
             "annotations": attrs["annotations"],
         }
+        +self._convert_seen_stats(attrs)
```

Comment:
This formatting is really weird - did black run here?

Reply:
I moved this addition to the other side of the dict...does that feel less awkward?